arch/arm/boot/dts/overlays/README

```diff
@@ -599,6 +599,16 @@ Load:   dtoverlay=audiosense-pi
 Params: <None>
 
 
+Name:   ugreen-dabboard
+Info:   Configures the ugreen-dabboard I2S overlay
+        This is a simple overlay based on the simple-audio-card and the dmic 
+        codec. It has the speciality that it is configured to use the codec 
+        as a master I2S device. It works for example with the Si468x DAB 
+        receiver on the uGreen DABBoard.
+Load:   dtoverlay=ugreen-dabboard
+Params: <None>
```

Comment:
There is a parameter - "card-name", so it must be documented here, including adding ",<param>=<val>" to the Load: line.